Source/WebCore/platform/graphics/gstreamer/TrackPrivateBaseGStreamer.cpp

```diff
@@ -290,8 +301,7 @@ void TrackPrivateBaseGStreamer::notifyTrackOfStreamChanged()
     if (!streamId)
         return;
 
-    GST_INFO("Track %d got stream start for stream %s.", m_index, streamId.get());
```

Comment:
Please, if you have the opportunity before merging the PR, add again this GST_INFO() line to the final patch. I find these log lines useful when having to debug track issues.
In order to do that, since the PR is already approved, you can just include this line in the commit message. No need to ask for an additional review:
```
Reviewed by Xabier Rodriguez-Calvar and Philippe Normand.
```
Thanks!

Reply:
That line should be included automatically by the commit queue.

Reply:
Even after resubmitting a patch with fixes? I thought the r+ were reset in that case. That's why I've always filled in the "Reviewed by" line when resubmitting last minute fixes after the patch was approved. Thanks for clarifying!

Reply:
That happened before github, since github it works.